### DISCUSSION TITLE
Bug with synctex

### DISCUSSION THREAD

**Discussion by vscodegithubby** [sumatrapdfreader/sumatrapdf#4631] [General]
I'm not sure if this is a bug with Sumatra, VSCode, or LaTeX Workshop.

When I read a PDF generated with LaTeX Workshop in VSCode, and the folder containing the PDF has a space in its name, double-clicking on the PDF in Sumatra should open the corresponding text in VSCode via SyncTeX. However, the space in the folder name causes VSCode to treat the spaces as separate files, attempting to open each one individually. Additionally, it adds the Sumatra.exe path to the list of "files."

It seems that Sumatra should enclose the path within quotes to handle spaces correctly.

**Comment by GitHubRulesOK** [comment]
check your return value has "%f"
![image](https://github.com/user-attachments/assets/9dd12653-e9df-4b5f-8c47-a031b772b0ff)


works for me
![image](https://github.com/user-attachments/assets/89037ab9-5e83-494a-bbfc-f2e74dfcc55a)
